author_api/models.py

```diff
@@ -30,9 +58,80 @@ def host(self):
     def url(self):
       return self.host + 'author/' + str(self.id)
 
+    objects = AuthorManager()
```

Comment:
This allows the custom queries to be used from AuthorQuerySet and AuthorManager. Basically it extends the Author.objects query manager to allow an areFriends query defined above.